app/controllers/api/v1/users_controller.rb

```diff
@@ -14,8 +14,8 @@ def index
 
   swagger_api :show do
     summary 'Responds with a user resource'
-    param :path, :user, :string, :required, "User Id"
-    response :ok, "Success", :User
+    param :path, :user, :string, :required, 'User Id'
```

Comment:
Is there a way we can lint the .rb files to catch this sort of thing before a checkin and manual review?